helm-charts/konk/values.yaml

```diff
@@ -63,6 +63,22 @@ etcd:
     # runAsNonRoot: true
     # runAsUser: 1000
 
+  # subchart options
+  subchart: true
+  auth:
+    rbac:
+      enabled: false
+    client:
+      secureTransport: true
+      existingSecret: # must override this with RELEASE_NAME-CHART_NAME-etcd-cert
```

Comment:
I'm not sure how to deal with this in the operator. We shouldn't have to leave it up to the user to define in their CR, since it could be constructed from the name of the CR.

Reply:
We might be able to flip it around: let the etcd chart generate the certs, and point to that secret in the apiserver deployment.